test/controllers/EmailControllerSpec.scala

```diff
@@ -109,12 +110,26 @@ class EmailControllerSpec extends ItSpec {
         doc.select("#newEmailInput").`val` shouldBe "somenewemail@newemail.com"
       }
     }
+
+    "throw an error if an email address cannot be found in the eligibility check response" in {
+      stubCommonActions()
+      EssttpBackend.TermsAndConditions.findJourney(isEmailAddressRequired = true, encrypter = testCrypto)(
+        JsonUtils.replace(List("AgreedTermsAndConditions", "eligibilityCheckResult", "customerDetails"), JsArray())(
```

Comment:
this seemed easier than going back through everything and adding the `customerDetails` as a parameter to loads of methods